internal/testutil/servicetest/orchestratortest/orchestrator.go

```diff
@@ -105,28 +105,6 @@ func NewTargetOfEvaluation(assuranceLevel string) *orchestrator.TargetOfEvaluati
 		toe.AssuranceLevel = &assuranceLevel
 	}
 
-	// Our test catalog does not allow scoping, so we need to emulate what we do in CreateTargetOfEvaluation
```

Comment:
I think the function "NewTargetOfEvaluationWithoutControlsInScope" and all the other MockControlsInScope* can be deleted.